lib/wikidata/fetcher.rb

```diff
@@ -23,16 +29,8 @@ def initialize(h)
       else
         raise 'No id'
       end
-      load_lookup_data!
-    end
-
-    LOOKUP_FILE = 'https://raw.githubusercontent.com/everypolitician/wikidata-fetcher/master/lookup.json'.freeze
-    def load_lookup_data!
-      lookup = JSON.load(
-        open(LOOKUP_FILE), nil, symbolize_names: true, create_additions: false
-      )
-      @@skip = lookup[:skip]
-      @@want = lookup[:want]
+      @@skip = self.class.lookup[:skip]
+      @@want = self.class.lookup[:want]
```

Comment:
Moved to methods in https://github.com/everypolitician/wikidata-fetcher/pull/35/commits/0868d11e64b4c5a368acbb1256f84b92a4e914a6